crates/opencascade-sys/build.rs

```diff
@@ -31,6 +31,5 @@ fn main() {
     println!("cargo:rustc-link-lib=static=wrapper");
 
     println!("cargo:rerun-if-changed=src/lib.rs");
-    println!("cargo:rerun-if-changed=cpp/wrapper.hxx");
```

Comment:
Instead of deleting this line, we can change it to the proper path:

`println!("cargo:rerun-if-changed=include/wrapper.hxx");`

Sorry I missed this in the original 😅

Reply:
Updated :)